blackbox/docs/interfaces/postgres.rst

```diff
@@ -153,30 +153,30 @@ For compatibility reasons there is a trimmed down `pg_type <pgsql_pg_type_>`__ t
 CrateDB::
 
     cr> select * from pg_catalog.pg_type order by oid;
-    +------+-------------+----------+---------+--------+--------------+--------------+---------+-----------+
-    |  oid | typbasetype | typdelim | typelem | typlen | typname      | typnamespace | typtype | typtypmod |
-    +------+-------------+----------+---------+--------+--------------+--------------+---------+-----------+
-    |   16 |           0 | ,        |       0 |      1 | bool         |    -68025646 | b       |        -1 |
-    |   18 |           0 | ,        |       0 |      1 | char         |    -68025646 | b       |        -1 |
-    |   20 |           0 | ,        |       0 |      8 | int8         |    -68025646 | b       |        -1 |
-    |   21 |           0 | ,        |       0 |      2 | int2         |    -68025646 | b       |        -1 |
-    |   23 |           0 | ,        |       0 |      4 | int4         |    -68025646 | b       |        -1 |
-    |  114 |           0 | ,        |       0 |     -1 | json         |    -68025646 | b       |        -1 |
-    |  199 |           0 | ,        |     114 |     -1 | _json        |    -68025646 | b       |        -1 |
-    |  700 |           0 | ,        |       0 |      4 | float4       |    -68025646 | b       |        -1 |
-    |  701 |           0 | ,        |       0 |      8 | float8       |    -68025646 | b       |        -1 |
-    | 1000 |           0 | ,        |      16 |     -1 | _bool        |    -68025646 | b       |        -1 |
-    | 1002 |           0 | ,        |      18 |     -1 | _char        |    -68025646 | b       |        -1 |
-    | 1005 |           0 | ,        |      21 |     -1 | _int2        |    -68025646 | b       |        -1 |
-    | 1007 |           0 | ,        |      23 |     -1 | _int4        |    -68025646 | b       |        -1 |
-    | 1015 |           0 | ,        |    1043 |     -1 | _varchar     |    -68025646 | b       |        -1 |
-    | 1016 |           0 | ,        |      20 |     -1 | _int8        |    -68025646 | b       |        -1 |
-    | 1021 |           0 | ,        |     700 |     -1 | _float4      |    -68025646 | b       |        -1 |
-    | 1022 |           0 | ,        |     701 |     -1 | _float8      |    -68025646 | b       |        -1 |
-    | 1043 |           0 | ,        |       0 |     -1 | varchar      |    -68025646 | b       |        -1 |
-    | 1184 |           0 | ,        |       0 |      8 | timestamptz  |    -68025646 | b       |        -1 |
-    | 1185 |           0 | ,        |    1184 |     -1 | _timestamptz |    -68025646 | b       |        -1 |
-    +------+-------------+----------+---------+--------+--------------+--------------+---------+-----------+
+    +------+----------+-------------+----------+---------+--------+--------------+--------------+---------+-----------+
```

Comment:
Is the output better readable if it is rendered? Otherwise we could maybe limit this to a couple columns since it is just here as an example and not as a thorough documentation. If we want to show which columns we have we could additionaly query `information_schema.columns` for the column names of `pg_catalog.pg_type`

Reply:
it is better readable, however, I'm not sure if this makes sense especially If we intend to add more columns. I'm in favor to limit it as you suggested.

@mfussenegger pushed a fixup